src/PrestaShopBundle/Resources/config/services/bundle/event_listener.yml

```diff
@@ -32,14 +32,6 @@ services:
         tags:
         - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 40 }
 
-    prestashop.multishop_command_listener:
```

Comment:
I need tests to check if this doesn't conflict with the PrestaShop application: this probably works as a side effect.